Source/Turbo/Navigator/Navigator.swift

```diff
@@ -76,6 +76,11 @@ public class Navigator {
         hierarchyController.clearAll(animated: animated)
     }
 
+    /// Reloads the main `Session`.
+    public func reload() {
```

Comment:
I'm wondering if we should reload the modal session too.

Reply:
Yeah good call - will do.